### ISSUE TITLE
fine-tune the massivekb-pretrained model with additional modifications

### ISSUE THREAD

**Issue by irleader** [Noble-Lab/casanovo#267]
Hi,

In the new release of v3.5.0,  "Specifying custom residues to retrain Casanovo is now possible" , so I would like to fine-tune/retrain the massivekb-pretrained model, is that possible?

In my new training dataset, there are two extra modficatioins, so I added them to config.yaml,  the total AA vocabulary size increases from 29 to 31. 

Usually decoder and final softmax layer's size needs to be adjusted accordingly, how can this be done in Casanovo v3.5.0? 

If not possible, can I replace 2 exisiting modifications with the extra 2 modifications I want (which keeps the AA vocabulary size 29 unchanged), and retrain the massivekb-pretrained model?

I am looking forward to your reply.

Best regards
